misk/src/main/kotlin/misk/web/actions/WebActions.kt

```diff
@@ -1,11 +1,14 @@
 package misk.web.actions
 
+import kotlinx.coroutines.CoroutineName
+import kotlinx.coroutines.runBlocking
 import misk.ApplicationInterceptor
 import misk.Chain
 import misk.web.HttpCall
 import misk.web.RealChain
 import kotlin.reflect.KFunction
 import kotlin.reflect.KParameter
+import kotlin.reflect.full.callSuspendBy
 
 fun WebAction.asChain(
```

Comment:
Aside: This function is only used in one place and it's internal, I wonder if it should be `internal`. It feels odd to be exposed as `public`. 

**This would break ABI compatibility.**